.maestro/browsing/omnibar_hint_behavior.yaml

```diff
@@ -0,0 +1,25 @@
+appId: com.duckduckgo.mobile.android
+name: "ReleaseTest: Multiple tabs can be opened"
+tags:
+    - releaseTest
+---
+- retry:
+      maxRetries: 3
+      commands:
+            - launchApp:
+                  clearState: true
+                  stopApp: true
+
+            - runFlow: ../shared/skip_all_onboarding.yaml
+
+            - assertVisible:
+                text: "Search"
+            - tapOn:
+                id: "omnibarTextInput"
+            - inputText: "https://privacy-test-pages.site"
+            - pressKey: Enter
+            - tapOn:
+                id: "omnibarTextInput"
+            - eraseText
+            - assertVisible:
+                text: "Search or type URL"
```

Comment:
does this need to exist? there is already `browsing/visit_site` which is also a `releaseTest` and it has a `tapOn: `search or type URL` step in it already

Reply:
I removed that test and updated `visit_site` instead 👍